es-vue-base/src/lib-components/EsFormInput.vue

```diff
@@ -130,6 +142,11 @@ export default {
             type: Boolean,
             default: false,
         },
+        phoneMaskValue: {
+            type: String,
+            default: '(###)-###-####',
```

Comment:
This should be `(###) ###-####` rather than `(###)-###-####`

Reply:
Based on the screenshot that Xuewei  provided for design it is (###)-###-###. However, I can make this like the one you suggested as that is general US Format and it makes sense for the default to a standard format.